javav2/example_code/transcribe/src/main/java/com/amazonaws/transcribestreaming/TranscribeMedicalStreamingDemoApp.java

```diff
@@ -0,0 +1,206 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Tests? At least simple integration tests like other Java examples.

Reply:
There is only one integration test for the transcribe examples, and it is a special case. This app uses voice input from a microphone, and that is difficult to simulate. Also, there are likely no microphone drivers on cloud instances.